apps/system-e2e/src/notifications/notify.ts

```diff
@@ -0,0 +1,15 @@
+import SlackNotify from 'slack-notify'
+;(async () => {
+  const webhookUrl = process.env.SLACK_WEBHOOK
+  if (webhookUrl) {
+    const slack = SlackNotify(webhookUrl)
+    slack
+      .send({
+        channel: '#acceptance-tests-status',
+        icon_emoji: ':boom:',
+        text: `The System E2E tests have failed, please check Spinnaker. The user that triggered the pipeline is: ${process.env.TRIGGER_USER}`,
```

Comment:
Bad DevX. Can this point to the nice playwright UI?

Reply:
I understand but - no, sorry. That UI is only available locally, we do not have it running.

Reply:
On second thought - it might be possible.